test/unit/accord/utils/Gens.java

```diff
@@ -433,16 +735,7 @@ public Gen<T[]> ofSizeBetween(int minSize, int maxSize) {
                 int size = sizeGen.nextInt(r);
                 T[] list = (T[]) Array.newInstance(type, size);
                 for (int i = 0; i < size; i++)
-                {
-                    try
-                    {
-                        list[i] = fn.next(r);
-                    }
-                    catch (IgnoreGenResult e)
-                    {
-                        return Arrays.copyOf(list, i);
-                    }
-                }
+                    list[i] = fn.next(r);
```

Comment:
I don't know when/why this was removed and looking at the `List` version this looks like a bug... we don't hit this case often with arrays which is prob why we didn't notice...

I plan to create a patch for cep-15-accord + accord to keep them in-sync with this PR... ill try to mixup in both places soon